operators/pkg/instance-controller/container_logic.go

```diff
@@ -256,6 +288,27 @@ func (r *InstanceReconciler) CreateContainerEnvironment(
 		return err
 	}
 
+	if !environment.Resources.Disk.IsZero() {
+		pvc := v1.PersistentVolumeClaim{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      name,
+				Namespace: namespace,
+			},
+		}
+
+		if _, err := ctrl.CreateOrUpdate(ctx, r.Client, &pvc, func() error {
+			// PVC's spec is immutable, it has to be set at creation
+			if pvc.ObjectMeta.CreationTimestamp.IsZero() {
+				pvc.Spec = buildContainerInstancePVCSpec(environment)
+			}
+			return ctrl.SetControllerReference(instance, &pvc, r.Scheme)
+		}); err != nil {
+			r.setInstanceStatus(ctx, "Could not create PVC "+pvc.Name+" in namespace "+pvc.Namespace+": "+err.Error(), "Error", "VmiNotCreated", instance, "", "")
+			return err
+		}
+		klog.Infof("Successfully created PVC for instance %s in namespace %s", instance, namespace)
```

Comment:
Nit. In the log, use the return value of CreateOrUpdate instead of the hard-coded `created` value, as in the example